modules/apps/frontend-js/frontend-js-web/src/main/resources/META-INF/resources/liferay/global.es.js

```diff
@@ -28,6 +28,7 @@ import {
 	getComponentCache,
 	initComponentCache,
 } from './component.es';
+import {confirm} from './confirm';
```

Comment:
One thing to consider is to place this util in `Modal.js` file, as it is basically a specialized variation of the default `openModal`.  In this way, it is quite similar to `openSelectionModal`. In that line, maybe we should even name it something like `openConfirmModal`, to have consistent naming.

Reply:
Yes, It makes sense. I like this idea of centralizing stuff inside one file but talking about my personal workflow, I like to CMD+P(open Files) -> type for the desired function/file that I was planning to see the implementation and for openModal stuff It required me some minutes of research to really understand where it was located but again, I'm fine with this :p

Reply:
What do you think about naming the utility `openConfirmModal`? On one side, it's more verbose and a bit clunky, but it would be consistent with `openModal`, `openSelectionModal` and `openPortletModal`. If a dev sees `open*Modal` it would imply that the implementation is in Clay/React modal. The same goes for alerts in https://github.com/liferay-frontend/liferay-portal/pull/1990, that util could be `openAlertModal`.

Reply:
> If a dev sees open*Modal it would imply that the implementation is in Clay/React modal

It makes sense. Updating...